src/web/collectors/CollectorsStatusPage.jsx

```diff
@@ -97,10 +98,12 @@ const CollectorsStatusPage = createReactClass({
   },
 
   _formatStatus(name, item) {
-    let restart = null;
+    let buttons = null;
     if (name !== 'Status' && this.state.collector) {
-      restart = (<div className="pull-right">
-        <CollectorsRestartButton collector={this.state.collector} backend={name}/>
+      buttons = (<div className="pull-right">
+        <CollectorsImportButton collector={this.state.collector} backend={name} />
```

Comment:
How about we only show this button if the sidecar is running a version that supports that (>= 0.1.8)?
Otherwise an message?